### PR TITLE
Real init...

### PR DESCRIPTION
**Pull request by xschildw**
Cloned from packer-rstudio, setup R 4.0 and shiny server instead of RStudio.
This is to prep AMI for ShinyServerPro.

### CODE REVIEW

**Comment by zaro0508** [comment]
i don't see travis build for this?

**Comment by xschildw** [comment]
It's on tci.com (https://travis-ci.com/github/Sage-Bionetworks-IT/packer-shinyserver). How did it end up there and not on .org?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# packer-base-rstudio
+# packer-base-shinyserver
```
Comment (zaro0508) [inline README.md:1]: I recommend copying some of the boilerplate readme info from [packer-ami-template README](https://github.com/Sage-Bionetworks-IT/packer-ami-template/blob/master/README.md)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      script:
+        - pre-commit autoupdate
+        - pre-commit run --all-files
+        - pushd src
+        - echo "ImageName = ${IMAGE_NAME}"
+        - packer validate -var ImageName=${IMAGE_NAME} template.json
+    - stage: deploy-branch
+      script:
+        - pushd src
+        - travis-wait-improved --timeout 45m packer build -force -var AwsProfile=${AWS_PROFILE} -var AwsRegion=${AWS_REGION} -var ImageName=${IMAGE_NAME} template.json
```
Comment (zaro0508) [inline .travis.yml]: add `-var PACKER_LOG=1`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - stage: deploy-tag
+      script:
+        - pushd src
+        - travis-wait-improved --timeout 45m packer build -force -var AwsProfile=${AWS_PROFILE} -var AwsRegion=${AWS_REGION} -var ImageName=${IMAGE_NAME} template.json
```
Comment (zaro0508) [inline .travis.yml]: add `-var PACKER_LOG=1`

File: src/template.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      "strip_path": true,
+      "type": "manifest"
+    }
+  ],
+  "provisioners": [
+    {
+      "ansible_env_vars": [
+        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'"
+      ],
+      "playbook_file": "playbook.yaml",
```
Comment (zaro0508) [inline src/template.json:48]: add 
```
      "extra_arguments": [
        "-v"
      ],
```

File: src/playbook.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+- hosts: all
+  become: yes
+
+  tasks:
+    # - name: Get base playbook
```
Comment (zaro0508) [inline src/playbook.yaml]: remove this commented section?

File: src/playbook.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    #   get_url: url=https://raw.githubusercontent.com/Sage-Bionetworks/packer-base-ubuntu-bionic/v1.0.0/src/playbook.yaml dest={{ playbook_dir }}/sagebio-base-ubuntu-bionic-v1.0.0.yaml
+
+    # - include_tasks: sagebio-base-ubuntu-bionic-v1.0.0.yaml
```
Comment (zaro0508) [inline src/playbook.yaml]: remove

**Review by zaro0508 — CHANGES_REQUESTED** [review]
i would recommend adding the .github folder with CODEOWNERS and PULL_REQUEST_TEMPLATE.md 
files, use the one from [packer-ami-template](https://github.com/Sage-Bionetworks-IT/packer-ami-template/blob/master/.github).

File: README.md (hunk 1 above)
Comment (xschildw) [reply README.md:1]: Yeah, I read the Confluence page after the fact...

File: src/playbook.yaml (hunk 5 above)
Comment (xschildw) [reply src/playbook.yaml]: Done

File: src/playbook.yaml (hunk 6 above)
Comment (xschildw) [reply src/playbook.yaml]: Done.

File: .travis.yml (hunk 3 above)
Comment (xschildw) [reply .travis.yml]: Done.

File: src/template.json (hunk 4 above)
Comment (xschildw) [reply src/template.json:48]: Done

**Review by tthyer — APPROVED** [review]
LGTM

**Review by zaro0508 — APPROVED** [review]

**Comment by zaro0508** [comment]
FYI, all builds for Sage-Bionetworks-IT org are in tci.com

**Review by jkiang13 — APPROVED** [review]
Looks good.

**Comment by xschildw** [comment]
Obsolete.

**Pull request closed without merging**